reconcile/change_owners/change_owners.py

```diff
@@ -372,9 +372,8 @@ def run(
             # e.g. gitlab-housekeeper rejects direct lgtm labels and the review-queue
             # skips MRs with this label
             if SELF_SERVICEABLE in labels:
-                gl.remove_label_from_merge_request(
-                    gitlab_merge_request_id, SELF_SERVICEABLE
-                )
+                merge_request = gl.get_merge_request(gitlab_merge_request_id)
+                gl.remove_label(merge_request, SELF_SERVICEABLE)
```

Comment:
Isn't that making two calls to GitLab API?

Reply:
Yes, it's same as before here https://github.com/app-sre/qontract-reconcile/pull/3702/files#diff-1d5ab1abb6ecb1d8c243a9e8c04d4cf6505e6be2c8932df654ea9a11c6c1b64bL453-L454, as `merge_request` is not fetched yet, will move this `get_merge_request` up when other gitlab methods also changed to accept `merge_request` as argument instead of `gitlab_merge_request_id`.